wade/chain.py

```diff
@@ -718,19 +732,34 @@ def special_op(self, peer_id, op_name, args, tag):
             peer_ids = [peer_id]
 
         resps = {}
+        resps_lock = threading.Lock()
```

Comment:
I accidentally tried do this without "dummy" and failed dramatically. :) The "dummy" part really matters! Python gets very upset when you try to spawn subprocesses with functions with complicated unpickleable objects. Also you can't spawn a subprocess on a function that's not importable outside the module i.e. anonymous functions like `make_request`. Anyway, the code is much cleaner now thanks for the recommendation.